tox.ini

```diff
@@ -63,3 +72,8 @@ exclude = .tox,build,dist,pysnooper.egg-info
 
 [pytest]
 addopts = --strict
+
+[pysnooper]
+commands =
+    pip install pysnooper
+    pip show pysnooper
```

Comment:
This is unneeded. You can remove it. (Explanation: The `[<tool_name>]` sections are configuration file sections that the tool reads. I don't think pysnooper reads anything out from `tox.ini`.)